src/migrate.rs

```diff
@@ -42,6 +42,7 @@ pub fn migrate(
         investment_denom: format!("{}.investment", old_state.raise),
         capital_denom: old_state.capital_denom,
         capital_per_share: old_state.capital_per_share,
+        fiat_deposit_addr: None,
```

Comment:
did we verify at a business level they won't really end up making us support existing contracts?

Reply:
I talked with Yarik yesterday, and he is trying to figure that out. We have funds coming up that want to use `usd.deposit` likely before the HL (finalization) flow will be ready. I told him migrating existing contracts and adding logic to support that in the SCs comes at a cost so I think he's trying to delay putting those on until we have support for `usd.deposit`. Long story short I believe we are leaning towards not supporting migration into this coin type.